epub/epub.d.ts

```diff
@@ -0,0 +1,66 @@
+// Type definitions for epub
+// Project: https://github.com/julien-c/epub
+// Definitions by: Julien Chaumond <https://github.com/julien-c>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+
+/**
+ *  new EPub(fname[, imageroot][, linkroot])
```

Comment:
This information should go on the `EPub` constructor in jsdoc format. Something like `@param {string} fname filename for the ebook` although the types are optional since they're already given by the constructor declaration.